apps/www/components/Frame/SecondaryNav.js

```diff
@@ -180,7 +234,6 @@ const styles = {
       '&.is-active': {
         ...fontStyles.sansSerifMedium,
         lineHeight: '16px',
-        marginTop: -1,
```

Comment:
This styling caused the active links to break when within a scroller.
The negative margin seemed very arbitrary to me so I removed it.
Visually I don't really see a difference, however I am unsure what the reason behind it was in the first place…